### ISSUE TITLE
bug: core card: key navigation focus issue on Internet Explorer

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#3562]
#### Is this a bug, enhancement, or feature request?

Bug

#### Briefly describe your proposal.

The tab navigation focus is not shown on the card content and click focus for keys (space/enter) is also no shown (buttons). The focus should show for tab navigation and click focus should be seen for click keys (space/enter)

browser: Internet Explorer 11
env: https://fundamental-ngx.netlify.app/#/core/card

#### If this is a bug, please provide steps for reproducing it.

1. use tab navigation on cards and watch the focus
2. use space key to select a button and watch for click focus
![Peek 2020-10-06 17-38](https://user-images.githubusercontent.com/47522152/95216494-cac41a80-07fa-11eb-83b8-ec92a9cebebf.gif)

**Comment by DeepakSap14** [comment]
fixing https://github.com/SAP/fundamental-ngx/issues/3553 will fix this issue.
